modules/nodes-pods-secrets-creating.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+[id='nodes-pods-secrets-creating_{context}']
```

Comment:
This module should be at least three smaller modules - the mod docs guide doesn't say, but it implies that there's only one procedure section per procedure module. It also implies that you can't have additional headings in the procedure.

Reply:
Done:
nodes-pods-secrets-creating
nodes-pods-secrets-creating-docker
nodes-pods-secrets-creating-opaque